README.md

```diff
@@ -497,35 +498,20 @@ Next you can verify that the currency contract has the proper initial balance:
 ```
 
 <a name="pushamessage"></a>
-### Pushing a message to a sample contract 
-
-To send a message to a contract you need to create a new user account who will be sending the message.
+### Pushing a message to sample "currency" contract 
 
-Firstly, generate the keys for the account:
-
-```bash
-cd ~/eos/build/programs/eosc/
-./eosc create key
-./eosc create key
-```
-
-And then create the `tester` account:
-
-```bash
-./eosc create account inita tester PUBLIC_USER_KEY_1 PUBLIC_USER_KEY_2 
-```
-
-After this you can send a message to the contract:
+Let's send a message from the **currency** account to account **inita** 
```

Comment:
Let's explain here that 'currency' account holds all the currency on creation and we can send some to another account?

Reply:
Yes

Reply:
This is mentioned in the previous step https://github.com/EOSIO/eos/pull/435/commits/fd09989e9cb31b02108a6af9c70d17215eabc2a2#diff-04c6e90faac2675aa89e2176d2eec7d8R490